Makefile.in

```diff
@@ -29,10 +29,10 @@ ES_IMAGE ?= docker.elastic.co/elasticsearch/elasticsearch-oss
 ES_IMAGE_TAG ?= 7.10.2
 
 SW_OAP_IMAGE ?= ghcr.io/apache/skywalking/oap
-SW_OAP_IMAGE_TAG ?= 8fad0cc8a822a3b266e528b30295e1376b956f1d
+SW_OAP_IMAGE_TAG ?= 48c96a233103ba869143062923d4033678b91f73
 
 SW_UI_IMAGE ?= ghcr.io/apache/skywalking/ui
-SW_UI_IMAGE_TAG ?= 8fad0cc8a822a3b266e528b30295e1376b956f1d
+SW_UI_IMAGE_TAG ?= 8bc67614681b37623fa64233f8389d6754e64cf2
```

Comment:
You are putting a wrong commit ID. The UI image is packaged in the main repository, not UI.

And you should verify this locally.

Reply:
This is dev stage image repo, https://github.com/apache/skywalking/pkgs/container/skywalking%2Fui

Reply:
OMG I made a stupid mistake, could we postpone the deployment? I'll fix it later

Reply:
We have fixed this manually

Reply:
Thank you for your trouble, I'll pay more attention in the future.